test/gtest/common/mem_buffer.cc

```diff
@@ -77,11 +77,25 @@ bool mem_buffer::is_rocm_managed_supported()
 {
 #if HAVE_ROCM
     int device_id, has_managed_mem;
-    return ((hipGetDevice(&device_id) == hipSuccess) &&
-            (hipDeviceGetAttribute(&has_managed_mem,
-                                   hipDeviceAttributeManagedMemory,
-                                   device_id) == hipSuccess) &&
-            has_managed_mem);
+    hipError_t ret;
+    void *dptr;
+    hipPointerAttribute_t attr;
+    if ((hipGetDevice(&device_id) == hipSuccess) &&
+        (hipDeviceGetAttribute(&has_managed_mem,
+                               hipDeviceAttributeManagedMemory,
+                               device_id) == hipSuccess) &&
+        has_managed_mem) {
+        ret = hipMallocManaged(&dptr, 64);
+        if (ret != hipSuccess)
+            return false;
+        ret = hipPointerGetAttributes(&attr, dptr);
```

Comment:
1. pls use { } for single-statement if 
2. pls add space line between the if statemets
for example
```c
        ret = ...
        if (ret != hipSuccess) {
             return false;
        }

        ret = hipPointerGetAttributes(&attr, dptr);
        ...
```